services/src/main/java/org/exoplatform/officeonline/AbstractOfficeOnlineService.java

```diff
@@ -282,6 +185,21 @@ protected StringBuilder platformUrl(String schema, String host, int port) {
     return platformUrl;
   }
 
+  /**
+   * Gets the user.
+   *
+   * @param username the username
+   * @return the user
+   */
+  protected User getUser(String username) {
```

Comment:
Where this method used? If it's about checking user presence, then better read from identity registry.

Reply:
WOPIService addUserMetadataProperties(...)

Reply:
OK, it's for user display name (friendly form for MS side).